app/components/finance/npq/participant_outcomes/table.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Finance
+  module NPQ
+    module ParticipantOutcomes
+      class Table < BaseComponent
+        attr_reader :participant_declaration
+
+        delegate :outcomes, to: :participant_declaration
+
+        def initialize(participant_declaration:)
+          @participant_declaration = participant_declaration
+        end
+
+        def render?
+          participant_declaration.participant_profile.npq?
+        end
+
+        def completion_date(outcome)
+          if outcome.voided?
+            outcome.updated_at.to_s(:govuk)
```

Comment:
voided outcomes still have a `completion_date`, maybe we can surface it as well?

Reply:
@cwrw 
Again this one is also specified in the ticket.

Maybe it makes sense to show the "created_at" as a new column, that way its clear when the outcome was "changed".

Reply:
same here, a bit confusing but if they understand this and prefer this set up then we can go with it